.github/workflows/main.yml

```diff
@@ -12,22 +12,20 @@ jobs:
     strategy:
       matrix:
         include:
-          - tox-env: py37
-            python-version: "3.7"
           - tox-env: py38
             python-version: "3.8"
           - tox-env: py39
             python-version: "3.9"
           - tox-env: pypy3
-            python-version: pypy3
+            python-version: "pypy3.9"
```

Comment:
This .github/workflows/main.yml is to test this cookiecutter only, not the generated project-slug right?  So do we even need all the different versions of python?

Reply:
I suppose not, but it's free, so why not.